Python.md

```diff
@@ -15,23 +20,63 @@ As most of the other projects written in Python, we use [PEP 8]
 (https://www.youtube.com/watch?v=wf-BqAjZb8M)" by Raymond Hettinger at PyCon 2015,
 which talks about the famous P versus NP problem.*
 
+If the project has a `pylintrc` file, you can use pylint to check for potential style violations:
+
+    pylint --rcfile=pylintrc
+
 The general idea is to make the code within a project consistent and easy to interpret (for humans).
 
+
 ### Docstrings
 
-Unless the function is easy to understand quickly, it should have a docstring
-describing what it does, how it does it, what the arguments are, and what the
-expected output is.
+Include docstrings in functions. This should say *what* the function does, its *inputs*,
+and *outputs* or possible *failure states* (exceptions).
```

Comment:
yes, there is a limit where we probably don't always need docstrings, but I'm a little worried that we'll have disagreements on what "easy to understand quickly" means - that is to say, I would argue that a function with a docstring is *always* easier to understand than reading code, regardless of length.
If a function is 1 line long and the name documents it, then perhaps documentation is not needed.